### ISSUE TITLE
get_assets_holdings has moved from reports to ops.holdings

### ISSUE THREAD

**Issue by mbafford** [hoostus/beancount-control-chart#1]
Thanks to: 
https://github.com/beancount/beancount/commit/29e0ead38ddae2bff8f8f2bf57089e0b13dbd9e0

Change seems to be simple, though:
```
-        raw_holdings_list, price_map = holdings_reports.get_assets_holdings(current_entries,
+        raw_holdings_list, price_map = holdings.get_assets_holdings(current_entries,
```

`holdings` is already imported, and `holdings_reports` no longer needs to be imported:

```
  from beancount.ops import holdings
  from beancount import loader
- from beancount.holdings import holdings_reports
```

with:

```beancount==2.3.1```

I'd submit a PR, but I'm not sure if this actually works - I'm getting no data on the chart when I run the script, but that may be more about the data in my journal. So I'm not 100% the above change works.

**Comment by hoostus** [comment]
Yes, that's the correct fix; it just needs to import from a different
place. I actually fixed this in my local version two or three weeks ago but
apparently forgot to push it to Github. Ooops.

**Comment by mbafford** [comment]
Thanks - I did realize that through my own testing last night. Here's the PR with a couple of other minor fixes as well.

Ultimately, the script isn't suited to my needs, but it gave me some good ideas. Thanks for putting it out there!

Found it from your [Google Groups post](https://groups.google.com/g/beancount/c/P-xpoOQRAQk/m/nBJf6FKpBgAJ) while looking for a way to calculate a moving average of an Expense category in beancount.
